### PR TITLE
Slack integration

### PR DESCRIPTION
**Pull request by F3licity**
Allow the WebScanner to send an output message to your preferred Slack channel.

### CODE REVIEW

**Comment by Basvanstein** [comment]
There are stilll todos in the code?

**Comment by F3licity** [comment]
@Basvanstein I've marked this PR ready for review. Can you have a look, when you can? Do we miss anything in order to push to the registry the new docker img?

**Comment by Basvanstein** [comment]
I still have a comment open on it.. for the rest it looks fine.

File: slackHandler.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+from slack_sdk.errors import SlackApiError
+
+logger = logging.getLogger(__name__)
+
+"""WebClient instantiates a client that can call API methods"""
+
+
+class SlackHandler:
+    def __init__(self, channel_id):
+        self.channel_id = channel_id
```
[the remark this answers is not in the document]
Comment (F3licity) [reply slackHandler.py:24]: added. can you have a look?

**Pull request merged**